orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/providers/cf/ServiceManifest.java

```diff
@@ -61,6 +62,7 @@ public Artifact getArtifact() {
   }
 
   @Data
+  @EqualsAndHashCode(callSuper = false)
```

Comment:
this was just a compiler warning and there are no fields on the superclass to consider